webapp/pages/GitHubRepositoryList.jsx

```diff
@@ -152,6 +152,14 @@ class GitHubRepositoryList extends AsyncPage {
     return github && github.scopes.indexOf('repo') !== -1;
   }
 
+  // TODO(tonyo): implement a nice and proper modal confirmation window
+  confirmDelete() {
+    const confirmMessage =
+      'Are you sure you want to delete this repository from Zeus?\n' +
+      'ALL data (build information, artifacts, etc.) will be removed.';
+    return window.confirm(confirmMessage);
+  }
```

Comment:
Haven't figured out yet a proper reactish way to have a nice confirmation/warning window :(